### ISSUE TITLE
Are there any way to provide Few-shot prompt in Prompty?

### ISSUE THREAD

**Issue by 07JP27** [microsoft/prompty#140]
I want to express Few-shot prompting in prompty file.
I think it can be achieved if I could write like the following Prompty format, but it seems that the 'assistant' role is not supported in Prompty.
Are there any way to provide few-shot in Prompty file or any schedule that implement assistant role in Prompty?

```prompty
---
metadata omitted
sample:
  question: What is Prompty?
---
system:
You are an AI assistant.

user:
[Few-shot user question]

assistant:
[Few-shot AI answer]

user:
{{question}}
```

**Comment by injeniero** [comment]
I see this working properly on the VS extension, and inspecting the python code, it seems it should work as well. 

What is not working is the "highlight" in the VS extension for the role assistant, but it still works.

**Comment by 07JP27** [comment]
@injeniero 
Thank you for your great information.
As you said, I assumed it was not supported because the assistant role was not highlighted in VS Code.
But it was indeed accomplished.
I hope this fact reflect to VS code extension (Highlight feature) and sample code in document of Prompty :)

```prompty
system:
You are an AI assistant supporting a user named "{{userName}}". 

user:
Hello

assistant:
Hi

user:
{{question}}
```

```python
prompt = prompty.load("./basic.prompty")
print(prompty.prepare(prompt, {"userName": "John", "question":"What's up?"}))
```

```text
[
  {'role': 'system', 'content': 'You are an AI assistant supporting a user named "John".'},
  {'role': 'user', 'content': 'Hello'}, 
  {'role': 'assistant', 'content': 'Hi'},
  {'role': 'user', 'content': "What's up?"}
]
```


 VS code view (only assistant role has not highlighted)
![Image](https://github.com/user-attachments/assets/ec0f08ef-491c-47b2-9c27-39b474171a2e)

**Comment by sethjuarez** [comment]
Looks like an extension issue. Will tag this as an extension bug.

**Comment by sethjuarez** [comment]
Yes! The `assistant:` role marker is fully supported in Prompty v2. You can write few-shot examples exactly as you described:

```prompty
---
name: few-shot-example
model:
  id: gpt-4
  provider: openai
---
system:
You are an AI assistant.

user:
What is the capital of France?

assistant:
The capital of France is Paris.

user:
{{question}}
```

The parser splits on `system:`, `user:`, and `assistant:` role markers to create the message list.

Documentation:
- **[File Format](https://prompty.ai/core-concepts/file-format/)** — role markers explained
- **[Pipeline](https://prompty.ai/core-concepts/pipeline/)** — how parsing works

Live at https://prompty.ai/
